lib/action.ml

```diff
@@ -317,6 +317,7 @@ module Action (Github_api : Api.Github) (Slack_api : Api.Slack) = struct
         Lwt.return_none
     in
     let process link =
+      log#info "processing links: %s" link;
```

Comment:
```suggestion
      log#debug "processing link: %s" link;
```

it's probably too noisy to log every link that is received, especially as a large number of those won't be github links